packaging/linux/buildpackage.sh

```diff
@@ -66,7 +66,11 @@ fi
 
 echo "Building core files"
 
-make version VERSION="${TAG}"
+if [ "${PACKAGING_OVERWRITE_MOD_VERSION}" == "True" ]; then
+    make version VERSION="${TAG}"
```

Comment:
Spaces instead of a tab here (and in the three other occurances below).

Reply:
Formatting was fixed.